src/icons/IconFood.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+type Props = { className?: string };
+
+export default function IconDelete({ className = '' }: Props) {
```

Comment:
IconDelete? The file name is IconFood.tsx

Also, have you noticed that there are lots of new icons in HDS now?

Reply:
My bad. Copy pasted to skeleton for it. Didn't find this one from HDS so I had to create it myself.